model/Dataset/Dataset.md

```diff
@@ -30,3 +30,19 @@ the following has to hold:
    `/Core/Relationship` of type `hasDeclaredLicense` having that element as its
    `from` property and a `/SimpleLicensing/AnyLicenseInfo` as its `to`
     property.
+
+## Summary @zh-Hans
+
+`Dataset` 配置文件基于 `Software` 配置文件提供了对数据集有用的附加元数据。
+
+## Description @zh-Hans
+
+`Dataset` 命名空间定义了与数据集相关的概念，包括其准备流程、特征和访问方法。
+
+## Profile conformance @zh-Hans
+
+元素集合要符合这个配置文件，必须满足以下条件：
+
+1.每一个 `/Dataset/DatasetPackage` 必须存在一个类型为 `hasConcludedLicense` 的 `/Core/Relationship` 元素，此元素作为它的 `from` 属性，`/SimpleLicensing/AnyLicenseInfo` 作为它的 `to` 属性。
+
+2.每一个  `/Dataset/DatasetPackage` 必须存在一个类型为 `hasDeclaredLicense` 的 `/Core/Relationship` 元素，此元素作为它的 `from` 属性， `/SimpleLicensing/AnyLicenseInfo` 作为它的 `to` 属性。
```

Comment:
```suggestion
1. 每一个 `/Dataset/DatasetPackage` 必须存在一个类型为 `hasConcludedLicense`
   的 `/Core/Relationship` 元素，此元素作为它的 `from` 属性，
   `/SimpleLicensing/AnyLicenseInfo` 作为它的 `to` 属性。
2. 每一个  `/Dataset/DatasetPackage` 必须存在一个类型为 `hasDeclaredLicense`
   的 `/Core/Relationship` 元素，此元素作为它的 `from` 属性，
   `/SimpleLicensing/AnyLicenseInfo` 作为它的 `to` 属性。
```

For Markdown to render the numbered list, we need a space after the numbers. (1., 2., ...).

Also, if possible, trying to put paragraphs within 80 characters limit, so it can be easily reviewed on the GitHub web UI (This is not mandatory, spec-parser and MkDocs are fine with very long paragraph. It's just only for a convenient of human reviewer).